<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerbookctgDialog</class>
 <widget class="QDialog" name="customerbookctgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0, 255, 0)</string>
  </property>
  <widget class="QLabel" name="label_gmailctg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>  gmail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eidctg">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>    eid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_identityctg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>154</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>   identity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gmailctg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>  Joey  @gmail.com</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_eidctg">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>    ctg01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_identityctg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>  book or booked</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updatecustomerctg">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
